tests/fixtures/poll.json

```diff
@@ -0,0 +1,63 @@
+{
+	"create_poll": {
+		"method": "POST",
+		"endpoint": "polls",
+		"data": {
```

Comment:
According to the docs, this returns a response that looks something like this

```
{
  "polls": [Poll]
}
```

Confirm this with a manual call to 2c.test and/or dev Canvas and adapt the fixture accordingly.